src/test/java/com/yelp/nrtsearch/server/grpc/LuceneServerTest.java

```diff
@@ -990,9 +990,7 @@ public void testBackupWarmingQueries() throws IOException, InterruptedException
                   .build());
       fail("Expecting exception on the previous line");
     } catch (StatusRuntimeException e) {
-      assertEquals(
-          "UNKNOWN: Unable to backup warming queries since uptime is 0 minutes, which is less than threshold 1000",
-          e.getMessage());
+      assertEquals("UNKNOWN: Unable to backup warming queries", e.getMessage().substring(0, 41));
```

Comment:
We do want to check that the error was because of less uptime. Can we create a regex pattern which accepts the error message but any number of minutes and do a pattern match instead of exact comparison? This would be something like:
```
Pattern pattern = Pattern.compile("UNKNOWN: Unable to backup warming queries since uptime is [0-9] minutes, which is less than threshold 1000");
boolean matchFound = pattern.matcher("UNKNOWN: Unable to backup warming queries").matches();
assertTrue(matchFound);
```

Reply:
@sarthakn7  great idea! I updated the PR to use pattern matching.  Please review again!  what are your thoughts?